data/asciidoctor.css

```diff
@@ -1,5 +1,6 @@
 /* Asciidoctor default stylesheet | MIT License | http://asciidoctor.org */
-@import "https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic|Noto+Serif:400,400italic,700,700italic|Droid+Sans+Mono:400";
+/* Remove the comments around the @import statement below when using this as a custom stylesheet */
+/*@import "https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400";*/
```

Comment:
This `@import` line should be uncommented when used in the Firefox add-on. The reason is the Firefox addon renders the document to embeddable HTML, which does not include the fonts. This is kind of a pain atm, but suffice to say, it's a necessary manual change.